query-service-impl/src/test/resources/application.conf

```diff
@@ -389,6 +412,16 @@ service.config = {
       requestHandlerInfo = {
         tenantColumnName: customer_id
         startTimeAttributeName = "EVENT.startTime"
+        additionalTenantFilters = [
+                  {
+                    "tenantId" = "__default"
+                    "timeRangeAndFilters" = [
+                      {
+                        "filter" : "{\r\n  \"lhs\": {\r\n    \"columnIdentifier\": {\r\n      \"columnName\": \"SERVICE.name\"\r\n    }\r\n  },\r\n  \"operator\": \"EQ\",\r\n  \"rhs\": {\r\n    \"literal\": {\r\n      \"value\": {\r\n        \"string\": \"dummyService\"\r\n      }\r\n    }\r\n  }\r\n}"
```

Comment:
Do we need `\r\n`? If we just put a single string, would it work?

Reply:
removed